recipes-core/rauc/files/rauc-mark-good

```diff
@@ -32,6 +32,26 @@ function info () {
     echo "$SCRIPT_NAME: $*" || true
 }
 
+# usb_boot - determine if the system has booted from a USB device
+# and appears to be a one time boot.
+#
+# 1) Device must be removable (or its not a USB)
+# 2) More than one niboota or nibootb exists (one of which is
+#    presumed to be on internal storage)
+#
+function usb_boot () {
+    # Check for 2 (or more) niboota or nibootb labeled partitions
+    [ `lsblk -lno NAME,PARTLABEL |grep niboota |wc -l` -gt 1 ] || return 1
+    [ `lsblk -lno NAME,PARTLABEL |grep nibootb |wc -l` -gt 1 ] || return 1
+
+    # is the device removable
+    devname=`mount |grep boot |cut -d" " -f1`
```

Comment:
STYLE: Could you prefer to use the newer `$()` subshell syntax over backticks? It's more consistent with our newer code and - I think - provides superior readability.

Reply:
Done.